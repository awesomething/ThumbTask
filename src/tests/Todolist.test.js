import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Todolist from '../components/Todolist'
import {configure} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

describe(`Todolist component`, () => {
  const props = {
    id: 'a',
    name: 'test-class-name',
    
  }

  it('renders a .Todolist by default', () => {
    const wrapper = shallow(<Todolist />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the Todolist given props', () => {
    const wrapper = shallow(<Todolist {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})

