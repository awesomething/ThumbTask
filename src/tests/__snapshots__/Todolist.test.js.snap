// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todolist component renders a .Todolist by default 1`] = `
<div
  className="todo-container"
>
  <ul
    className="todo-list"
  />
</div>
`;

exports[`Todolist component renders the Todolist given props 1`] = `
<div
  className="todo-container"
>
  <ul
    className="todo-list"
  />
</div>
`;
